plugins/ua_network_tcp.c

```diff
@@ -24,10 +24,6 @@
 # ifndef __clang__
 #  include <malloc.h>
 # endif
-/* Fix redefinition of SLIST_ENTRY on mingw winnt.h */
```

Comment:
I also wondered the same, but it worked without the push macro. Anyways it's better to use it as you already implemented: https://github.com/open62541/open62541/commit/987c910efd13ed3fa1a03bb7f6e21a6e6de425fe